entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/Organization.java

```diff
@@ -1,123 +1,293 @@
 package eu.europeana.entitymanagement.definitions.model;
 
+import java.lang.reflect.Field;
+import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
-import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import static eu.europeana.entitymanagement.vocabulary.WebEntityFields.*;
+
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.annotation.JsonSetter;
+
+import eu.europeana.entitymanagement.vocabulary.EntityTypes;
+
+/**
+ * This class defines base organization type of an entity.
+ */
+
+@JsonInclude(value = JsonInclude.Include.NON_EMPTY)
+@JsonPropertyOrder({CONTEXT, ID, TYPE, DEPICTION, PREF_LABEL, ACRONYM, ALT_LABEL, DESCRIPTION, FOAF_LOGO, EUROPEANA_ROLE,
+		ORGANIZATION_DOMAIN, GEOGRAPHIC_LEVEL, COUNTRY, FOAF_HOMEPAGE, FOAF_PHONE, FOAF_MBOX, HAS_ADDRESS, IDENTIFIER, SAME_AS})
+public class Organization extends Entity {
+
+	public Organization() {
+		super();
+		// TODO Auto-generated constructor stub
+	}
+
+	private Map<String, String> description;
+	private Map<String, List<String>> acronym;
+	private String logo;
+	private String homepage;
+	private List<String> phone;
+	private List<String> mbox;
+	private Map<String, List<String>> europeanaRole;
+	private Map<String, List<String>> organizationDomain;
+	private Map<String, String> geographicLevel;
+	private String country;
+	private Map<String, String> countryMap;
+
+	//address fields
+	private String hasAddress;
```

Comment:
the hasAddress field should be an Address object (the class exists in the code, this is also legacy part related to solr representation. see also Organization.jsonld in the ticket: https://europeana.atlassian.net/browse/EA-964)

Reply:
The deleted AddressImpl should be restored as Address class